src/utils/stringUtils.ts

```diff
@@ -220,3 +220,13 @@ export const formatName = (firstName?: string, lastName?: string): string => {
   }
   return "";
 };
+
+/**
+ * Parses a number of seconds into a string formatted as "mm:ss".
+ *
+ * @note Supports a maximum of 59 minutes and 59 seconds.
+ * @param seconds The number of seconds to convert.
+ * @returns A string formatted as "mm:ss".
+ */
+export const secondsToMinuteString = (seconds: number) =>
```

Comment:
This works well. An optional alternative solution could be using dayjs. Super not required, just thought I'd bring it up.

```javascript
import dayjs from "dayjs";
import duration from "dayjs/plugin/duration";

// Extends it globally. Normally this would be in a config file,
// but looks like we don't have one for dayjs
dayjs.extend(duration);

export const secondsToMinuteString = (seconds: number) =>
  dayjs.duration(seconds, "seconds").format("mm:ss");
```

Reply:
I migrated this util to our dateUtils file (it makes more sense there IMO) and updated it to DayJs in e68bcdd.